### PR TITLE
first tasks. HNML

### PR DESCRIPTION
**Pull request by kirsmi**
The task was Long because my computer was broken

### CODE REVIEW

File: ksmiahlikau/01.HTML/Task_2.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+			<ul><li>Наша первая программа</li>
+			<li>Пример цикла For</li></ul>
+			<!--текст абзаца.-->		    
+			<P>Из множества существующих языков программирования Python — один из самых лёгких в изучении. Он создан в конце 80-х годов XX века, но с тех пор он сильно изменился. Хотя Python входит в состав большинства дистрибутивов Linux, на него часто не обращают внимания при выборе языка для изучения. В этой статье мы рассмотрим создание консольных приложений, а в следующей — программирование графических интерфейсов (GUI). Давайте создадим простую программу прямо сейчас.</P>		 
+			<!--заголовок-->		
+			<H2>Наша первая программа</H2>		
+			<!--текст-->		
+			<P>Наберём код в текстовом редакторе (например, в gedit). Потом мы разберёмся, что делает каждая строка и пойдём дальше.</P>
+			<P>Введите эти 4 строки:</P>
+			<P><code><!--код программы-->
```
Comment (pluhin) [inline ksmiahlikau/01.HTML/Task_2.html:28]: Do not use tag p for not human text

File: ksmiahlikau/01.HTML/Task_2.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+				#!/usr/bin/env python<br>
+				print 'Hello. I am a python program.'<br>
+				name = raw_input("What is your name?")<br>		
+				print "Hello there, " + name + "!"<br></code></P>			
+			<P>Это всё. Сохраните файл как hello.py в любом месте. Я предлагаю сохранить его в папке
+				python_examples в домашнем каталоге. Этот пример показывает, как просто написать программу на
+				Python. Для запуска программы нужно сделать её файл исполняемым. Для этого введите в терминале</P>
+				<!--код программы-->
+			<p><code>chmod +x hello.py</code></p>
```
Comment (pluhin) [inline ksmiahlikau/01.HTML/Task_2.html:37]: Sorry, it was hackers

File: ksmiahlikau/01.HTML/Task_2.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+			<p>в папке, в которой вы сохранили программу. Теперь запустите её.</p>
+			<p><code>greg@earth:~/python_examples$ ./hello.py<br>
```
Comment (pluhin) [inline ksmiahlikau/01.HTML/Task_2.html:39]: Do not use tag p for not human text

File: ksmiahlikau/01.HTML/Task_2.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+			<p>в папке, в которой вы сохранили программу. Теперь запустите её.</p>
+			<p><code>greg@earth:~/python_examples$ ./hello.py<br>	
+				Hello. I am a python program.<br>	
+				What is your name? Ferd Burphel<br>	
+				Hello there, Ferd Burphel!<br>	
+				greg@earth:~/python_examples$</code></p>	
+			<p>Вот и всё.</p>		
+			<H2>Пример цикла For</H2>	
+			<p>Теперь давайте рассмотрим пример программирования цикла. Запустите текстовый редактор и наберите следующую программу:</p>
+			<p><code><pre>
```
Comment (pluhin) [inline ksmiahlikau/01.HTML/Task_2.html:47]: Do not use tag p for not human text
and shift tag code into tag pre

File: ksmiahlikau/01.HTML/Task_3.html [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+			<ul><li><u>Наша первая программа</u></li>
+			<li><u>Пример цикла For</u></li></ul>
+			<!--текст абзаца.-->		    
+			<P>Из множества существующих языков программирования Python — один из самых лёгких в изучении. Он создан в конце 80-х годов XX века, но с тех пор он сильно изменился. Хотя Python входит в состав большинства дистрибутивов Linux, на него часто не обращают внимания при выборе языка для изучения. В этой статье мы рассмотрим создание консольных приложений, а в следующей — программирование графических интерфейсов <u>(GUI)</u>. Давайте создадим простую программу прямо сейчас.</P>		 
+			<!--заголовок-->		
+			<H2>Наша первая программа</H2>		
+			<!--текст-->		
+			<P>Наберём код в текстовом редакторе (например, в <b>gedit</b>). Потом мы разберёмся, что делает каждая строка и пойдём дальше.</P>
+			<P>Введите эти 4 строки:</P>
+			<P><code><!--код программы-->
```
Comment (pluhin) [inline ksmiahlikau/01.HTML/Task_3.html:28]: Do not use tag p for not human text

File: ksmiahlikau/01.HTML/Task_3.html [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+				#!/usr/bin/env python<br>
+				print 'Hello. I am a python program.'<br>
+				name = raw_input("What is your name?")<br>		
+				print "Hello there, " + name + "!"<br></code></P>			
+			<P>Это всё. Сохраните файл как <b>hello.py</b> в любом месте. Я предлагаю сохранить его в папке
+				<b>python_examples</b> в домашнем каталоге. Этот пример показывает, как просто написать программу на Python. Для запуска программы нужно сделать её файл исполняемым. Для этого введите в терминале</P>
+				<!--код программы-->
+			<p><code><u><strong><i>chmod +x hello.py</i></strong></u></code></p>
```
Comment (pluhin) [inline ksmiahlikau/01.HTML/Task_3.html:36]: Do not use tag p for not human text

File: ksmiahlikau/01.HTML/Task_3.html [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+			
+			<p>в папке, в которой вы сохранили программу. Теперь запустите её.</p>
+			<p><code>greg@earth:~/python_examples$ ./hello.py<br>
```
Comment (pluhin) [inline ksmiahlikau/01.HTML/Task_3.html:39]: Do not use tag p for not human text

File: ksmiahlikau/01.HTML/Task_3.html [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+			<p>в папке, в которой вы сохранили программу. Теперь запустите её.</p>
+			<p><code>greg@earth:~/python_examples$ ./hello.py<br>	
+				Hello. I am a python program.<br>	
+				What is your name? Ferd Burphel<br>	
+				Hello there, Ferd Burphel!<br>	
+				greg@earth:~/python_examples$</code></p>	
+			<p>Вот и всё.</p>		
+			<H2>Пример цикла For</H2>	
+			<p>Теперь давайте рассмотрим пример программирования цикла. Запустите текстовый редактор и наберите следующую программу:</p>
+			<p><code><pre>
```
Comment (pluhin) [inline ksmiahlikau/01.HTML/Task_3.html:47]: Do not use tag p for not human text
and tag code move into pre

**Review by pluhin — CHANGES_REQUESTED** [review]

File: ksmiahlikau/01.HTML/Task_2.html (hunk 2 above)
Comment (kirsmi) [reply ksmiahlikau/01.HTML/Task_2.html:37]: very interesting =)

File: ksmiahlikau/01.HTML/Task_2.html (hunk 1 above)
Comment (kirsmi) [reply ksmiahlikau/01.HTML/Task_2.html:28]: ok fixed

File: ksmiahlikau/01.HTML/Task_2.html (hunk 3 above)
Comment (kirsmi) [reply ksmiahlikau/01.HTML/Task_2.html:39]: ok fixed

File: ksmiahlikau/01.HTML/Task_2.html (hunk 4 above)
Comment (kirsmi) [reply ksmiahlikau/01.HTML/Task_2.html:47]: ok fixed

File: ksmiahlikau/01.HTML/Task_3.html (hunk 5 above)
Comment (kirsmi) [reply ksmiahlikau/01.HTML/Task_3.html:28]: ok fixed

File: ksmiahlikau/01.HTML/Task_3.html (hunk 7 above)
Comment (kirsmi) [reply ksmiahlikau/01.HTML/Task_3.html:39]: ok fixed

File: ksmiahlikau/01.HTML/Task_3.html (hunk 8 above)
Comment (kirsmi) [reply ksmiahlikau/01.HTML/Task_3.html:47]: ok fixed

**Pull request merged**